### DISCUSSION TITLE
Editing the tailwindcss config in Next.JS project

### DISCUSSION THREAD

**Discussion by discoverlance-com** [projen/projen#1670] [Q&A]
I created a new next.js projen project with typescript but I am not able to edit the tailwind config file.

How do we edit the config file?

And by default the project adds `import "tailwindcss/tailwind.css"` to the index page to add tailwindcss but this fails and results in an error, I am assuming it's from the config not being setup properly but then also, in the tailwindcss package, there's no tailwind.css file so that could also be the issue.

The Error: 

```
error - ./node_modules/next/dist/build/webpack/loaders/css-loader/src/index.js??ruleSet[1].rules[2].oneOf[7].use[1]!./node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js??ruleSet[1].rules[2].oneOf[7].use[2]!./node_modules/tailwindcss/tailwind.css
TypeError: Cannot read properties of undefined (reading 'config')
Could not find files for / in .next/build-manifest.json
Could not find files for / in .next/build-manifest.json
(node:21228) [DEP_WEBPACK_MODULE_ISSUER] DeprecationWarning: Module.issuer: Use new ModuleGraph API
(Use `node --trace-deprecation ...` to show where the warning was created)
```

**Comment by Chriscbr** [comment]
> How do we edit the config file?

I don't think there is a way to directly edited right now - pull requests are welcome. We would basically need to expose `tailwindOptions` from https://github.com/projen/projen/blob/main/src/web/postcss.ts within [NextJsProject](https://github.com/projen/projen/blob/main/src/web/next.ts).

Regarding the error, I think this would be better created as an issue, as it could be a bug with the default configuration

**Reply by petehanssens** [reply]
Hey can we somehow get this fixed?
